recipe/migrations/python39.yaml

```
@@ -11,6 +11,10 @@ __migrator:
             - 3.9.* *_cpython   # new entry
             - 3.6.* *_73_pypy
     paused: false
+    exclude:
+      # this shouldn't attempt to modify the python feedstocks
+      - python
+      - pypy3.6
```

```suggestion
      - pypy3.6
      - pypy-meta
```

do we want this too?

---

probably can't hurt